code/modules/cargo/bounties/assistant.dm

```diff
@@ -141,6 +141,26 @@
 	reward = 3000
 	wanted_types = list(/obj/item/organ/tail/lizard)
 
+/datum/bounty/item/assistant/lizard_tail_whip
+	name = "Liz o' Nine Tails"
+	description = "Commander Jackson is looking for a rather unusual addition to her exotic weapons collection. She will reward you handsomely for a Liz o' Nine Tails."
+	reward = 4000
+	wanted_types = list(/obj/item/melee/chainofcommand/tailwhip)
+	exclude_types = list(/obj/item/melee/chainofcommand/tailwhip/kitty)
+
+/datum/bounty/item/assistant/cat_tail
+	name = "Cat Tail"
+	description = "Central Command has run out of heavy duty pipe cleaners. Can you ship over a cat tail to help us out?"
+	reward = 3000
+	wanted_types = list(/obj/item/organ/tail/cat)
+
+/datum/bounty/item/assistant/cat_tail_whip
+	name = "Cat o' Nine Tails"
+	description = "Commander Jackson is looking for yet another fine addition to her exotic weapons collection. She will reward you handsomely for a Cat o' Nine Tails."
+	reward = 4000
+	wanted_types = list(/obj/item/melee/chainofcommand/tailwhip/kitty)
+	exclude_types = list(/obj/item/melee/chainofcommand/tailwhip)
```

Comment:
I believe the `exclude_types` here and on the Firefighter are unnecessary.

Reply:
I'd at least keep the one on the Firefighter bounty so crew can't cash in with a much cheaper Ripley.